orchestrator/gorc/src/commands/deploy/erc20.rs

```diff
@@ -1,11 +1,15 @@
 use crate::{application::APP, prelude::*};
 use abscissa_core::{Clap, Command, Runnable};
 use ethereum_gravity::deploy_erc20::deploy_erc20;
+use ethers::prelude::*;
 use gravity_proto::gravity::{DenomToErc20ParamsRequest, DenomToErc20Request};
-use gravity_utils::connection_prep::{check_for_eth, create_rpc_connections};
+use gravity_utils::{
+    connection_prep::{check_for_eth, create_rpc_connections},
+    ethereum::downcast_to_u64,
+};
 use std::convert::TryFrom;
 use std::process::exit;
-use std::time::Duration;
+use std::{sync::Arc, time::Duration};
```

Comment:
I wonder why the Arc dependence is needed here. It's not obviously being used anywhere.

Reply:
The EthClient type is an alias of `Arc<SignerMiddleware<<Provider<Http>, LocalWallet>>>`